### DISCUSSION TITLE
Downgrade Pandas Version

### DISCUSSION THREAD

**Discussion by AmriDomas** [teamalgoritma/community#649] [Q&A] [answered]
Dear All,

<img width="530" alt="image" src="https://user-images.githubusercontent.com/130373441/236625621-95486ed4-1eaf-4a1d-8670-868378a88c19.png">

di anaconda prompt pandas sudah saya downgrade ke 1.3.4
<img width="289" alt="image" src="https://user-images.githubusercontent.com/130373441/236625784-7a91e9ff-86f3-4eea-aff9-634513f79880.png">

tetapi di jupyter notebook masih versi 2.0.0...

solusinya seperti apa?

**Comment by idoali** [comment]
Terima kasih atas pertanyaannya pak

Apa sudah dipastikan anaconda prompt dan jupyter notebook yang bapak tampilkan berasal dari environment yang sama pak?

**Comment by AmriDomas** [comment]
Environment masih sama

**Comment by idoali** [comment]
Pastikan juga selain virtual environment, kernelnya juga sudah tepat.

**Comment by ccalvocm** [comment, marked as answer]
Salam @AmriDomas , Anda perlu menambahkan lingkungan Anda dengan pandas 1.3.4 ke jupyter, untuk melakukannya buka terminal dan ketik:

`conda activate YourEnvironmentHere`
`python -m ipykernel install --user --name=YourEnvironmentHere`
 
Sekarang Anda akan memiliki lingkungan conda di notebook Jupyter, Anda dapat memilih YourEnvironmentHere di daftar kernel.

@AmriDomas

**Comment by nadyadtm** [comment]
“Halo @AmriDomas , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.”
